apps/greencheck/management/commands/dump_green_domains.py

```diff
@@ -23,62 +36,141 @@ def get_conn_string(self):
         # change this if we ever leave mysql, obvs
         return f"mysql://{db['USER']}:{db['PASSWORD']}@{db['HOST']}/{db['NAME']}"
 
-    def export_to_sqlite(self, database_url: str, db_name: str):
-        return subprocess.run(
-            ["db-to-sqlite", database_url, db_name, "--table=greendomain"]
+    @classmethod
+    def export_to_sqlite(cls, database_url: str, db_path: str) -> None:
+        """
+        Export the `cls.TABLE` to `db_path` from given `database_url`.
+
+        :param database_url: The database URL to export the `cls.TABLE` from.
+        :param db_path: The path to file to export SQLite database to.
+        """
+        cls._subprocess(
+            ["db-to-sqlite", database_url, db_path, f"--table={cls.TABLE}"],
+            f'Failed to export the "{cls.TABLE}" table to "{db_path}" SQLite database.'
         )
 
-    def prepare_for_datasette(self, db_name: str):
+    @classmethod
+    def prepare_for_datasette(cls, db_path: str) -> None:
```

Comment:
I've realised that this entire method is no longer needed now. It used to make sure that indexes used by sqlite were there, but we now use the same tables to serve requests, so this whole `prepare_for_datasette` method could be delete here, and also in the call to `handle`.

Would you mind removing it this class, and from where it is used in `handle` please

Reply:
Sure